src/recipe/admin/src/runtime/components/Message.vue

```diff
@@ -14,15 +14,16 @@
       height="1.5rem"
       width="100%"
     />
-    <span v-else-if="data">{{ data }}</span>
+    <span v-else-if="data">{{ l(data) }}</span>
```

Comment:
when localizing data we use `requireLocalization` flag from schema. see how we used localization in data of `Select` and `SelectButton`